packages/webpack/test/e2e-hack.spec.js

```diff
@@ -55,3 +55,20 @@ test('webpack/hack/fetch - cannot pollute prototypes', (t) => {
     { message: /Cannot add property polluted, object is not extensible/ }
   )
 })
+
+test('webpack/hack/mismatched-curlies', async (t) => {
+  t.plan(1)
+  try {
+    t.context.build = await scaffold({
+      ...webpackConfigDefault,
+      entry: {
+        hack4: './hack4.js',
+      },
+    })
+  } catch (err) {
+    t.assert(
+      err.compilationErrors[0].message.startsWith('E_VALIDATION'),
+      'Expected a validation error from erapper check'
```

Comment:
```suggestion
      'Expected a validation error from wrapper check'
```

Reply:
(referring to `packages/webpack/src/buildtime/wrapper.js`)

Reply:
I wouldn't mind if you pushed up a commit fixing that without waiting for anyone else.